moped-database/metadata/tables.yaml

```diff
@@ -736,17 +736,17 @@
   - name: moped_proj_components_subcomponents
     using:
       foreign_key_constraint_on:
+        column: project_component_id
         table:
-          name: moped_proj_components_subcomponents
           schema: public
-        column: project_component_id
+          name: moped_proj_components_subcomponents
   - name: moped_proj_features
     using:
       foreign_key_constraint_on:
```

Comment:
same with these, since the reshuffling seems like it may impact the way things are nested?

Reply:
the nesting is the same, just shuffled around